pkgs/top-level/all-packages.nix

```diff
@@ -8949,8 +8949,6 @@ with pkgs;
 
   idle3tools = callPackage ../tools/system/idle3tools { };
 
-  ifcopenshell = with python3Packages; toPythonApplication ifcopenshell;
```

Comment:
my understanding is that this is useful if python provides some cli tools. This is not the case here. 

There *are* some binaries in `$out/bin` though. I believe that to make things properly, these should be in another derivation in libraries/, with python build disabled. Or maybe this just uses the current default python3 version and in this case I should keep this line?